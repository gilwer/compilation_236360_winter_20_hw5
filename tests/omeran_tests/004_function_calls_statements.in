// calling functions as statements
enum days { SUN, MON, TUES };

int foo(int a) {
  printi(a);
  return a;
}

void kaki(int a, int bb, int c) {
  int x = a + bb + c;
  printi(x);
}

void bar() {
  print("bar");
  print("bar again");
  return;
}

void main() {
  int x = 9;
  kaki(foo(3), x, 4); // 3, 16
  foo(2); // 2
  foo(x); // 9
  foo(44+33); // 77
  foo (x * 15); // 135
  bar(); // bar, bar again
}