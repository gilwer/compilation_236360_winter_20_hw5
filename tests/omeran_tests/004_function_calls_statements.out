3
16
2
9
77
135
bar
bar again
